src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/SupplementaryEvidenceTest.java

```diff
@@ -132,29 +128,13 @@ private boolean hasCaseBeenUpdatedWithSupplementaryEvidence(
         return updatedScannedDocuments.size() == excpectedScannedDocuments && evidenceHandled.equals("No");
     }
 
-    private void verifySupplementaryEvidenceDetailsUpdated(int expectedScannedDocuments) throws JSONException {
-
-        JSONObject updatedCaseData = updateEnvelope(
-            "envelopes/supplementary-evidence-envelope.json",
-            caseDetails.getId()
-        );
-
-        Envelope updatedEnvelope = EnvelopeParser.parse(updatedCaseData.toString());
-        List<ScannedDocument> scannedDocumentList = getScannedDocuments(newEnvelope);
-
-        List<ScannedDocument> scannedDocumentList2 = getScannedDocuments(updatedEnvelope);
-        if (!scannedDocumentList2.isEmpty()) {
-            scannedDocumentList.addAll(scannedDocumentList2);
-        }
-
+    private void verifySupplementaryEvidenceDetailsUpdated(int expectedScannedDocuments) {
         assertThat(updatedScannedDocuments).isNotEmpty();
         assertThat(updatedScannedDocuments.size()).isEqualTo(expectedScannedDocuments);
-        assertThat(scannedDocumentList.size()).isEqualTo(expectedScannedDocuments);
-        assertThat(scannedDocumentList).containsAll(updatedScannedDocuments);
```

Comment:
making assertions on a list that was built a few lines above by loading a hardcoded json file, replacing some fields in it, and adding new elements.